hbase-shell/src/test/ruby/hbase/admin2_test.rb

```diff
@@ -435,5 +435,39 @@ def teardown
       end
     end
   end
+
+  # Tests for the `regioninfo` shell command
+  class RegionInfoTest < Test::Unit::TestCase
+    include TestHelpers
+    include HBaseConstants
+
+    def setup
+      setup_hbase
+    end
+
+    def teardown
+      shutdown
+    end
+
+    define_test "Get region info without any args" do
+      assert_raise(ArgumentError) do
+        command(:regioninfo)
+      end
+    end
+
+    define_test 'Get region info with encoded region name' do
+      output = capture_stdout { command(:regioninfo, '1588230740') }
```

Comment:
How do we know that `1588230740` region exists? Can we create a table, list_regions and then get regioninfo with this command?

Reply:
Yeah, it would be better. 
Will fix later, Thanks.